cobalt/build/gn.py

```diff
@@ -96,19 +96,22 @@ def write_build_args(build_args_path, original_lines, dict_settings,
 
 
 def main(out_directory: str, platform: str, build_type: str,
-         overwrite_args: bool, gn_gen_args: List[str]):
+         gn_gen_args: List[str]):
   platform_path = f'cobalt/build/configs/{platform}'
   dst_args_gn_file = os.path.join(out_directory, 'args.gn')
   src_args_gn_file = os.path.join(platform_path, 'args.gn')
   Path(out_directory).mkdir(parents=True, exist_ok=True)
 
-  if overwrite_args or not os.path.exists(dst_args_gn_file):
-    build_args = get_build_args(src_args_gn_file)
-    write_build_args(dst_args_gn_file, build_args[0], build_args[1], build_type)
-  else:
-    print(f'{dst_args_gn_file} already exists.' +
-          ' Running ninja will regenerate build files automatically.')
-
+  build_args = get_build_args(src_args_gn_file)
```

Comment:
nit: move this to be right before the `write_build_args` command as it's not used in the below conditional

Reply:
and another nit: I prefer unpacking the value when you get it so you can have named variables instead of a tuple you have to index into

Reply:
did the first one , for the second one im planning on doing more cleanup in https://github.com/youtube/cobalt/pull/4427, can address it there